lib/modules/manager/osgifeature/extract.ts

```diff
@@ -0,0 +1,81 @@
+import * as json5 from 'json5';
```

Comment:
In the README for this feature I see that the OSGi compendium is on release `8`. What happens when OSGi releases version `9`, with breaking changes?

Maybe the code should check which version of OSGi it uses, and throw an error if it's a newer version which we don't support?

Reply:
The implementations referenced in the OSGI specs rarely make breaking changes.

There is a provision for versioning at https://docs.osgi.org/specification/osgi.cmpn/8.0.0/service.feature.html#d0e156846, but no actual implementation uses it. The versioning part works by declaring an attribute of the form

```
"feature-resource-version": "1.0",
```

I can read it, and fail if it exists and does not "semver-match" `^1`.  Does that sound ok?

Reply:
It sounds good to me! 😉